package.json

```diff
@@ -64,6 +64,7 @@
     "@types/storybook__react": "^4.0.2",
     "@typescript-eslint/eslint-plugin": "^1.13.0",
     "@typescript-eslint/parser": "^1.13.0",
+    "core-js": "^2.4.0",
```

Comment:
This package is required for storybook, otherwise it will throw an error and will not start.

Reply:
Thanks for pointing this out. In https://github.com/iov-one/ponferrada/pull/684 there is a better solution for the problem (using less dependencies)